lib/interviewer/selectors/interface.js

```diff
@@ -133,7 +140,7 @@ export const makeNetworkNodesForPrompt = () => getNetworkNodesForPrompt;
 */
 
 export const getNetworkNodesForOtherPrompts = createSelector(
-  getNetworkNodesForType, getPropPromptId,
+  getNetworkNodesForType, getPromptId,
```

Comment:
As above, update to use the session state selector rather than looking for prompt ID in props.